proposals/UX.md

```diff
@@ -0,0 +1,45 @@
+# Application form for starting a SIG
+
+## What will be the name of the SIG
+
+User Experience SIG:  **UX-SIG**
+![Creativity](https://positivepsychology.com/wp-content/uploads/boost-your-creativity.jpg)
+
+## Which two persons will act as SIG Lead?
+
+- Jesus Garcia Gonzalez
+- Sven van der Burg
+
+
+## What is the SIG's mission?
+
+<!-- The mission of your SIG should contribute to the organization's mission. -->
+
+The SIG aims to start slowly expanding the knowledge and the importance of Human-centric design and best User Experience techniques within our organization and research projects, especially when developing software. 
+Visualization and presentation are also substantial parts of the SIG, improving seamless guidelines and keeping an eye on consistency and best practices following the eScience branding and communication guidelines. During the SIG meetings, the focus will be on providing design critiques to existing projects, analyzing the design and procedures, and giving feedback to the team members.
+
+## What GitHub repository will the SIG use?
+To be created
+
+## What Office group will the SIG use?
+UX-SIG, ux@esciencecenter.nl (to be created)
+
+## Plans for period until the end of April 2020
+
+- **Design Critiques for current projects**:  encouraging a positive culture to improve projects. Analyze the design of the project's user journey, usability, accessibility, interaction, visualization, and presentation, and give feedback to team members. While making sure of keeping  a **Clear scope for the conversation",   "Agreed-upon design objectives for the work."  and motivating "Conversation rather than command." One project per Meeting. 
+
+- **Proportionate UX tools and resources**: from tools for ideation, wireframing and prototyping, to resources for best practices and design patterns.
+ 
+## What are the expected outputs of the proposed SIG?
+
+- **Design Thinking workshops**:  a Design Thinking workshop is a hands-on, activity-based session built around the Design Thinking process.
```

Comment:
Is this something else than the 'Design Critqiues for current projects'? Or is it the same? And what is the outcome of these workshops? I.e. what do we gain from it?

I would propose something like: During the Design Critique sessions we will make NLeSC projects significantly more user-centered, thereby assuring that NLeSC projects bring more value

Through the bring your own project design critique session

Reply:
You are right, I am now realizing that this doesn't have to be _per se_ a task for the SIG, but an initiative for a different group, so we can keep the focus on Design Critiques. I removed the point from the document.